source/libs/dom-utils.ts

```diff
@@ -45,3 +45,9 @@ export const wrapAll = (targets: Element[], wrapper: Element): void => {
 	targets[0].before(wrapper);
 	wrapper.append(...targets);
 };
+
+export const isEditable = (target: Element | EventTarget | null): boolean => {
```

Comment:
```suggestion
export const isEditable = (node?: node): boolean => {
```

Reply:
@bfred-it It doesn't work: https://travis-ci.org/sindresorhus/refined-github/builds/547586732

```
ERROR in /home/travis/build/sindresorhus/refined-github/source/features/esc-to-deselect-line.tsx
[tsl] ERROR in /home/travis/build/sindresorhus/refined-github/source/features/esc-to-deselect-line.tsx(16,15)
      TS2345: Argument of type 'EventTarget | null' is not assignable to parameter of type 'Node | undefined'.
  Type 'null' is not assignable to type 'Node | undefined'.
ERROR in /home/travis/build/sindresorhus/refined-github/source/features/selection-in-new-tab.tsx
[tsl] ERROR in /home/travis/build/sindresorhus/refined-github/source/features/selection-in-new-tab.tsx(8,52)
      TS2345: Argument of type 'EventTarget | null' is not assignable to parameter of type 'Node | undefined'.
  Type 'null' is not assignable to type 'Node | undefined'.
```

Reply:
`node?: any`, since it’s a type guard function

Maybe the question mark isn’t necessary

Reply:
Without the `?` it works.